client/signup/steps/domains/index.jsx

```diff
@@ -1421,8 +1422,7 @@ export class RenderDomainsStep extends Component {
 				const [ content, sideContent ] = this.getContentColumns();
 
 				return (
-					<AsyncLoad
-						require="./async-domain-step-wrapper"
+					<StepContainerV2DomainsStep
```

Comment:
I only created `async-domain-step-wrapper` to facilitate the async loading. I think it'd be better to roll this into the main domain step component, since all it does now is prop drilling.

Can be a follow up though.